pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+# extend-exclude needs to be a regular expression
+extend-exclude = "third_party/"
+target-version = ["py37", "py38", "py39", "py310"]
```

Comment:
Just out of curiosity: is the target-version for Python here required? I just would like to reduce the duplicate work to add/remove Python versions in a lot of places if possible.

Reply:
It not specified, black will assume the python version it is executed in, which may affect compatibility.